manager/integration/tests/test_recurring_job.py

```diff
@@ -753,3 +743,12 @@ def test_recurring_jobs_when_volume_detached_unexpectedly(settings_reset, set_ra
     pod_names = common.get_deployment_pod_names(core_api, deployment)
 
     assert read_volume_data(core_api, pod_names[0], 'default') == data
+
+    # cleanup the backupstore so we don't impact other tests
+    # since we crashed the engine replica initiated the backup,
+    # it's backupstore lock will still be present, so we need
+    # to wait till the lock is expired, before we can delete
+    # the backups
+    vol.recurringUpdate(jobs=[])
+    backupstore.backupstore_wait_for_lock_expiration()
+    backupstore.backupstore_cleanup(client)
```

Comment:
Will the test fixture do backupstopre_cleanup?
If so we only need to wait for lock expiration.

Reply:
I wasn't sure if backupstore_wait_for_lock_expiration looks confusing on its own in the case. Thanks for the feedback, I've removed it and updated the in-code comment.